views/js/qtiCreator/widgets/interactions/hottextInteraction/states/Map.js

```diff
@@ -138,15 +139,15 @@ define([
         });
 
         _.forEach(interaction.getChoices(), function(choice){
-            var $score;
             var id = choice.serial + '-score';
+            var inputVal = typeof mapEntries[choice.id()] !== 'undefined' ? mapEntries[choice.id()] : '';
             var $hottext = $('[data-serial="' + choice.serial + '"]', $container);
+            
             if($hottext.length){
-
-                $score = $("<input type='text' name='" + id + "' class='score' data-validate='$numeric' data-validate-option='$allowEmpty; $event(type=keyup)' />");
-                $score.val(typeof mapEntries[choice.id()] !== 'undefined' ? mapEntries[choice.id()] : '');
-                $hottext.append($score);
-
+                $hottext.append(scoreTpl({
+                     id : id,
```

Comment:
Please check your ESLint, the indentation of these 2 lines is 21 spaces and you have some trailing whitespace above on L145.

Reply:
Also please check the copyright year of the file.

Reply:
fixed